src/app/containers/InlineLink/index.new.test.jsx

```diff
@@ -11,19 +14,30 @@ const fragmentBlock = (text, attributes = []) => ({
   },
 });
 
+const testInternalInlineLink = (description, locator, blocks) => {
+  shouldMatchSnapshot(
+    description,
+    /*
+      for the value it would bring, it is much simpler to wrap a react-router Link in a Router, rather than mock a Router or pass come mocked context.
```

Comment:
```suggestion
      for the value it would bring, it is much simpler to wrap a react-router Link in a Router, rather than mock a Router or pass some mocked context.
```